wye/base/constants.py

```diff
@@ -67,3 +67,11 @@ class WorkshopRatings:
 class WorkshopAction:
     ACTIVE = ('active', 'deactive')
     ASSIGNME = ('opt-in', 'opt-out')
+
+
+@choices
+class ContactUsType:
```

Comment:
There is already a FeedbackType, so I am renaming it to ContactFeedbackType. @vnbang2003